cli/golem-templates/templates/ts/ts-app-common/package.json

```diff
@@ -20,6 +19,6 @@
     "rollup": "^4.29.1",
     "tslib": "^2.8.1",
     "typescript": "^5.8.3",
-    "@rttist/typegen": "^0.2.0"
+    "@golemcloud/golem-ts-typegen": "^0.0.1-dev.5"
```

Comment:
Replaced RTTIST with golem-ts-typegen.
RTTIST was in the final bundle,  but zero things about reflection in final bundle